### PR TITLE
Beta Release 0.8.12

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
**Features**
- Generalization of Glance
  - Glance component
  - Text Glance component
    - Birthday Glance as example

**Notes**
Happy birthday @smartwhatt!

### CODE REVIEW

File: public/static/locales/th/home.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "glance": {
+    "birthday": "<b>วันนี้{{name}}ครบรอบอายุ {{year}} ปี!</b> อย่าลืมเฉลิมฉลองกันล่ะ"
```
Comment (smartwhatt) [inline public/static/locales/th/home.json:3]: This sound very google translated.

Not sure how to make it more natural

**Review by smartwhatt — DISMISSED** [review]
LGTM and TYSM!!! 🙏🙏

**Comment by SiravitPhokeed** [comment]
@smartwhatt I revised the translation and also fixed an issue with the Seat Dialog. Have another look.

**Review by smartwhatt — APPROVED** [review]
LGTM!!!

**Pull request merged**